module-sys/SystemManager/CpuSentinel.cpp

```diff
@@ -28,11 +28,12 @@ namespace sys
     void CpuSentinel::HoldMinimumFrequency(bsp::CpuFrequencyMHz frequencyToHold)
     {
         if (currentFrequencyToHold != frequencyToHold) {
-            auto msg = std::make_shared<sys::HoldCpuFrequencyMessage>(GetName(), frequencyToHold);
+            auto msg = std::make_shared<sys::HoldCpuFrequencyMessage>(GetName(), frequencyToHold, xTaskGetCurrentTaskHandle());
```

Comment:
Is it safe to send raw FreeRTOS task handles? Is it possible to receive an invalid handle in a different thread context, for instance during system shutdown? I'm just thinking out loud.